docs/api/synthetics/monitors/add-monitor-api.asciidoc

```diff
@@ -0,0 +1,168 @@
+[[add-monitor-api]]
+== Add Monitor API
+++++
+<titleabbrev>Add Monitor</titleabbrev>
+++++
+
+Creates a new monitor with the specified attributes. A monitor can be one of the following types: HTTP, TCP, ICMP, or Browser. The required and default fields may vary based on the monitor type.
+
+=== {api-request-title}
+
+`POST <kibana host>:<port>/api/synthetics/monitors`
+
+`POST <kibana host>:<port>/s/<space_id>/api/synthetics/monitors`
+
+=== {api-prereq-title}
+
+You must have `all` privileges for the *Synthetics* feature in the *{observability}* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>.
+
+[[monitor-request-body]]
+==== Request Body
+
+The request body should contain the attributes of the monitor you want to create. The required and default fields differ depending on the monitor type:
+
+*Common Fields*:
+
+`name`:: (Required, string): The monitor's name.
+
+`type`:: (Required, string): The monitor type (e.g., "http," "tcp," "icmp," or "browser").
+
+`schedule`:: (Required, object): The monitor's schedule.
+
+  * `number` (Optional, string, default: '3'): The number of intervals.
+
+  * `unit` (Optional, string, default: "minutes"): The unit of the schedule.
+
+:private-locations-url: https://www.elastic.co/guide/en/observability/current/synthetics-private-location.html
+:locations-command: https://www.elastic.co/guide/en/observability/current/synthetics-command-reference.html#elastic-synthetics-locations-command
+
+`locations`:: (https://github.com/elastic/synthetics/blob/main/src/locations/public-locations.ts#L28-L37[`Array<SyntheticsLocationsType>`])
+Where to deploy the monitor. Monitors can be deployed in multiple locations so that you can detect differences in availability and response times across those locations.
++
+To list available locations you can:
++
+  * Run the https://www.elastic.co/guide/en/observability/current/synthetics-command-reference.html#elastic-synthetics-locations-command[`elastic-synthetics locations`] command
```

Comment:
You mentioned that part of the reason for exposing a public api is that users don't want to have to fuss around with a node environment. This documentation for fetching list of available public and private locations still requires a node environment and installation of the @elastic/synthetics . Perhaps we should have a way in the UI to expose the options and document that here instead or in addition to.